ntcore/src/main/java/edu/wpi/first/networktables/NetworkTableType.java

```diff
@@ -118,8 +118,9 @@ public static String getStringFromObject(Object data) {
     } else if (data instanceof Float) {
       return "float";
     } else if (data instanceof Long) {
+      // Checking Long because NT supports 64-bit integers
       return "int";
-    } else if (data instanceof Double || data instanceof Number) {
```

Comment:
Was this done to be a catch-all for other classes that extend the `Number `class?

Reply:
Yes

Reply:
In that case, I will revert that